docs: Add ppa term entry (#60)
canonical/open-documentation-academy: landscape/explanation/glossary/ppa.md

# PPA 

> A **Personal Package Archive (PPA)** is a software repository specifically made for Ubuntu users. PPAs are used by developers to distribute their software. When you install Landscape, you provide a PPA so your system knows where to download Landscape and its required packages. For example, if you want the beta version of Landscape, you must add the Landscape beta PPA to your system.

Reviewer: ```suggestion
A **Personal Package Archive (PPA)** is a software repository specifically made for Ubuntu users. PPAs are used by developers to distribute their software. When you install Landscape, you provide a PPA so your system knows where to download Landscape and its required packages. For example, if you want the beta version of Landscape, you must add the Landscape beta PPA to your system.

```

# PPA 

> **Personal Package Archive(PPA)** is a software repository specifically made for Ubuntu-based products. This tool allows developers to create their own repositories that they can use to distribute their own software. In the context of Landscape, the PPA contains the tool's software for the server. 

Reviewer: ```suggestion
A **Personal Package Archive (PPA)** is a software repository specifically made for Ubuntu users. PPAs are used by developers to distribute their software. When you install Landscape, you provide a PPA so your system knows where to download Landscape and its required packages. For example, if you want the beta version of Landscape, you must add the Landscape beta PPA to your system.
```

Reviewer: Note: I did minimize the second sentence here a bit - your information wasn't wrong, but I just felt it might be confusing for Landscape users if we described PPAs in such a general sense. You are correct that developers can create their own repos and distribute their own software with PPAs, but since Landscape uses specific PPAs owned by Canonical, I changed that part to be more broad.

> It's important to use the correct PPA when you install Landscape. This is because there are different PPAs for different versions of Landscape 

Reviewer: ```suggestion
It's important to use the correct PPA when you install Landscape. This is because there are different PPAs for different versions of Landscape. For example, the Landscape beta PPA is `ppa:landscape/self-hosted-beta`, and the PPA for Landscape 24.04 LTS is `ppa:landscape/self-hosted-24.04`. Before you install Landscape, make sure you select the desired PPA for your system.
```

> . For example, the Landscape beta PPA is `ppa:landscape/self-hosted-beta`, and the PPA for Landscape 23.03 LTS is `ppa:landscape/self-hosted-23.03`. Before you install Landscape, make sure you select the desired PPA for your system.

Reviewer: ```suggestion
```

> Think of Landscape's PPA as picking a movie or show from your playlist on your favorite streaming site. You get to pick specific software from the PPA based on the needs of your software's repository. 

Reviewer: ```suggestion

```

Reviewer: I decided to remove this analogy because I don't think one is necessary here with all the other helpful info you've included. :smiley: 

But also (and this is only because I know Landscape well), I don't think this analogy is quite accurate - I think a more accurate analogy with your context would be that Landscape PPA is a streaming service (e.g., Amazon Prime Video) where you want to watch exclusive content (e.g., An Amazon Prime Video-only show, such as Fallout). So, in order to get the Landscape software (Fallout), you need to first have its location (Amazon).

That's just for your own knowledge though! No need to refactor that into an analogy for the documentation.
